SshAgentLib/Util.cs

```diff
@@ -44,6 +46,27 @@ public static class Util
 
     static string assemblyTitle;
 
+    /// <summary>
+    /// Convert a SecureString to an asni string in the form of a PinnnedArray&lt;byte&gt;
+    /// </summary>
+    /// <param name="ss"></param>
+    /// <param name="func"></param>
+    /// <typeparam name="T"></typeparam>
```

Comment:
Looks like there need to be deleted.

Reply:
I removed those lines.

Reply:
It doesn't look like these changes got pushed.